### PR TITLE
Added the lesson 3 directory as well as the slicing lab file.

### PR DESCRIPTION
**Pull request by avalvillar**
First lab for lesson 3 complete.

### CODE REVIEW

**Comment by avalvillar** [comment]
Submitting the second lab for lesson 3

File: students/avalvillar/lesson3/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#!/usr/bin/env python3
+
+#print("Starting Series 1:")# -> Used during coding/testing
+fruits = ["Apples", "Pears", "Oranges", "Peaches"] #New List
+print(fruits)
+#Ask the user to provide a fruit and place it at the end of the list.
+new_fruit = input("What fruit would you like to add to the list? > ")
+fruits.append(new_fruit)
+print(fruits)
+number_choice = input("Give me a number? (1-5) > ") #Which index (fruit) removed.
```
Comment (charlesrobison) [inline students/avalvillar/lesson3/list_lab.py]: What if there are more than 5 fruits in the list?

File: students/avalvillar/lesson3/list_lab.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+item = 0 #Counter used to go through list while manipulating.
+while item < len(fruits):
+    like = input("Do you like " + fruits[item].lower() + "? > ")
+    while like.lower() != "yes" and like.lower() != "no": #Yes or No is needed - loop
+        like = input("Do you like " + fruits[item].lower() + "? (yes or no) > ")
+    if like.lower() == "no": #If no then remove the item and continue
+        fruits.remove(fruits[item])
+    else:
+        item += 1 #If yes then increment and check the next index
+print("The list of fruits you like is: " + str(fruits))
```
Comment (charlesrobison) [inline students/avalvillar/lesson3/list_lab.py]: Nice!

File: students/avalvillar/lesson3/slicing_lab.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+assert front_back_every_other(a_tuple) == (), "assert front_back_every_other failed"
+assert front_back_every_other(a_list) == [4, 6], "assert front_back_every_other failed"
+
+assert reverse(a_string) == 'gnirts a si siht', "assert reverse failed"
+assert reverse(a_tuple) == (32, 5, 12, 13, 54, 2), "assert reverse failed"
+assert reverse(a_list) == [10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0], "assert reverse failed"
+
+assert mid_last_first(a_string) == "is a stringthis ", "assert mid_last_first failed"
+assert mid_last_first(a_tuple) == (13, 12, 5, 32, 2, 54), "assert mid_last_first failed"
+assert mid_last_first(a_list) == [3, 4, 5, 6, 7, 8, 9, 10, 0, 1, 2], "assert mid_last_first failed"
```
Comment (charlesrobison) [inline students/avalvillar/lesson3/slicing_lab.py]: Looks good - nice assert statements!

**Pull request merged**